winterfell/SETUP_WINTERFELL.md

```diff
@@ -1,58 +1,48 @@
 # Setting up Winterfell
 Winterfell is the server that handles all control logic in the application. This includes authentication, handling user information, it is basically the bulk of the main logistics.
 
-### Install Node
-This project is designed to work with node.js v5. 
+### Install Node Modules
+This project is designed to work with node.js v4.x.
 
 If you haven't already, we'll need to install node. Run the following at the same directory level as _package.json_.
+
 ```
+$ brew install node
 $ npm install
 ```
 
-### Install MongoDb No-SQL Database
-Refer to the [MongoDb installation page](https://docs.mongodb.com/master/installation/) to install.
-We used Homebrew to install Mongo.
-```
-$ brew install mongodb
-```
+### Startup Vagrant
 
-### Install Redis In-Memory DataStore
-We used NPM to install [Redis](http://redis.io/).
 ```
-$ npm install redis
+$ vagrant up
 ```
 
-## Running Winterfell locally
-Make sure a _MongoDb_ instance is started in a separate terminal, we assume the database is on port 27017.
-```
-$ mongod
-```
-or
-```
-$ ./start-mongod.sh
-```
+### Run Tests
 
-Make sure a _Redis_ instance is started in a separate terminal, we assume the database is on port 6379.
-```
-$ redis-server
-```
-or
 ```
-$ ./start-redis.sh
+$ vagrant ssh
+$ cd /vagrant/winterfell
+$ ./run-tests.sh
 ```
 
-Simply run the _start.sh_ script from the command line. The server should start up at port 3999.
+### Run Server
+
 ```
+$ vagrant ssh
```

Comment:
same here.
Let's have a script run-server.sh in the winterfell directory that calls
vagrant ssh
./vagrant/winterfell/start-server.sh

This keeps us at our current working directory.